packages/lib/src/services/player-sdk.service.ts

```diff
@@ -126,28 +127,36 @@ export class PlayerSdk {
     if (name === 'ready') {
       this.callbackStore.storeCallback(`event:${name}`, callback);
 
-      this.readyPromise.then(() => {
-        // check if callback was not removed
-        if (this.callbackStore.getCallbacks(`event:${name}`).includes(callback)) {
-          callback();
-        }
-      });
+      this.readyPromise
+        .then(() => {
+          // check if callback was not removed
+          if (this.callbackStore.getCallbacks(`event:${name}`).includes(callback)) {
+            callback();
+          }
+        })
+        .catch((err) => {
```

Comment:
I had to add this catch to cater for https://typescript-eslint.io/rules/no-floating-promises/